lib/Runtime/Math/WasmMath.h

```diff
@@ -17,6 +17,8 @@ class WasmMath
     template<typename T> static T Nearest(T aLeft);
     static int Rol(int aLeft, int aRight);
     static int Ror(int aLeft, int aRight);
+    template <typename T, T MIN, T MAX> static bool IsInValidRange (double aLeft);
```

Comment:
Hmm, wondering if this is in the right place. IsInValidRange isn't actually exposed through Wasm.